Packs/FireEyeHX/Integrations/FireEyeHXv2/FireEyeHXv2.py

```diff
@@ -1614,6 +1614,17 @@ def get_indicator_conditions(client: Client, args: Dict[str, Any]) -> CommandRes
     )
 
 
+def validation_baseurl(baseurl: str) -> None:
+    error_message = ''
```

Comment:
```suggestion
    # any of the folloiwng combinations is not allowed as suffix: /v3, /api/v3, /hx/api/v3 etc. 
    # The error message is built to include the complete suffix that should be removed (rather than running 2 or 3 times, seeing an error each time)
    error_message = ''
```